src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -1183,12 +1183,24 @@ void LpNormCompute(const nnvm::NodeAttrs& attrs,
   } else {
     small = ReduceAxesShapeImpl(inputs[0].shape_, param.axis, true, false);
   }
+
+  std::cout << "safe_accumulation: " << dmlc::GetEnv("MXNET_SAFE_ACCUMULATION", false) << std::endl;
```

Comment:
pls remember to remove it after all tests are passed

Reply:
Done.